scripts/update.sh

```diff
@@ -84,18 +85,15 @@ setup() {
     pip install -r requirements.txt
   fi
 
-  # Remove the oxa-tools-config directory if it exists to avoid the error
-  # "fatal: destination path 'oxa-tools-config' already exists and is not an empty directory."
-  if [[ -d $OXA_TOOLS_PATH/oxa-tools-config ]]; then
-    sudo rm -R $OXA_TOOLS_PATH/oxa-tools-config
-  fi
+  if [[ ! -d $OXA_PATH/oxa-tools-config ]]; then
+    cd $OXA_PATH
 
-  # Fetch the latest secrets from the private repo via a personal access token
-  cd $OXA_TOOLS_PATH
-  sudo git clone https://$ACCESS_TOKEN@github.com/microsoft/oxa-tools-config.git
+    # Fetch the latest secrets from the private repo via a personal access token
+    sudo git clone -b master https://$ACCESS_TOKEN@github.com/microsoft/oxa-tools-config.git
```

Comment:
Fine for now, but once we add release branch we'll want to separate the clone and checkout commands